wifi-qr

```diff
@@ -349,6 +340,22 @@ scan_connect_hidden() {
   nmcli con up "$QSSID" >/dev/null 2>&1
 }
 
+check_for_package() {
```

Comment:
Its good for single bash script. 
we will implement it.
- PPA or upstream is no need and not allowed. 

WiFi-QR was remove sudo check. 
Enduser will be use GUI default shortcut.
please check .desktop and .policy 
- sudo 
- pkexec

other suggestion, we fix and megre in master branch. 
also make single-script branch ?